modules/index-api/src/main/java/org/apache/ignite/internal/idx/InternalSortedIndex.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.idx;
+
+import java.util.BitSet;
+import java.util.List;
+import org.apache.ignite.internal.schema.Column;
+import org.apache.ignite.internal.util.Cursor;
+import org.apache.ignite.lang.IgniteUuid;
+import org.apache.ignite.table.Tuple;
+
+/**
+ * Sorted index facade.
+ */
+public interface InternalSortedIndex {
+    /** Exclude lower bound. */
+    byte GREATER = 0;
+
+    /** Include lower bound. */
+    byte GREATER_OR_EQUAL = 1;
+
+    /** Exclude upper bound. */
+    byte LESS = 0;
+
+    /** Include upper bound. */
+    byte LESS_OR_EQUAL = 1 << 1;
+
+    /**
+     * Return index name.
+     *
+     * @return Index name.
+     */
+    String name();
+
+    /**
+     * Return indexed table.
+     *
+     * @return Indexed table.
+     */
+    String tableName();
```

Comment:
why do we need tableName if we already have tableId ? I check all usages of such a method and seems we can obtain tableName by id there.

Reply:
removed